sentinelsat/__init__.py

```diff
@@ -3,5 +3,5 @@
 # Import for backwards-compatibility
 from . import sentinel
 
-from .sentinel import SentinelAPI, SentinelAPIError, InvalidChecksumError, read_geojson,\
-    geojson_to_wkt, format_query_date
+from .sentinel import InvalidChecksumError, SentinelAPI, SentinelAPIError, SentinelAPILTAError, \
```

Comment:
Should we maybe start using `__all__` in `sentinel.py` to select the imports?

Reply:
While I agree, maybe we should leave this for a separate edit/PR. It might get too confusing.